scss/bootstrap/overrides/style/_navs.scss

```diff
@@ -55,6 +55,18 @@
   }
 }
 
+.nav-pills {
+  margin-bottom: 15px;
```

Comment:
In our design `navs-pills` have the margin bottom, so it should be overridden right? Because throughout our design all nav-pills would have this `margin-bottom`